app/controllers/market_places_controller.rb

```diff
@@ -1,12 +1,23 @@
 class MarketPlacesController < ApplicationController
-  before_action :require_login, except: %i[index, show]
+  before_action :require_login, except: [:index]
```

Comment:
could you run rubocop on this file to check offenses?

usually, rubocop suggest `%i[index]` over `[:index]`